lib/ravendb/documents/commands/hilo_next_command.rb

```diff
@@ -1,24 +1,26 @@
 module RavenDB
   class HiloNextCommand < RavenCommand
     def initialize(tag, last_batch_size, last_range_at, identity_parts_separator, last_range_max)
-      super("", Net::HTTP::Get::METHOD)
+      super()
       @tag = tag
       @last_batch_size = last_batch_size
       @last_range_at = last_range_at
       @last_range_max = last_range_max
       @identity_parts_separator = identity_parts_separator
     end
 
-    def create_request(server_node)
-      @params = {
+    def create_request(server_node, url:)
```

Comment:
Just FYI I dropped the URL output parameter from `createRequest()` in Node.js client. It appears it's not used anywhere. I think you can safely remove it as well

Reply:
Done.